ReactAndroid/src/main/java/com/facebook/react/views/text/TextAttributeProps.java

```diff
@@ -609,6 +614,12 @@ private void setAccessibilityRole(@Nullable String accessibilityRole) {
     }
   }
 
+  private void setTextAlignVertical(@Nullable String alignVertical) {
+    if (alignVertical != null) {
+      mVerticalAlign = alignVertical;
+    }
+  }
+
```

Comment:
to be avoided when view isVirtual otherwise we add ReactAlignSpan to parent TextView